openmc/cmfd.py

```diff
@@ -527,116 +525,160 @@ def run_adjoint(self, run_adjoint):
-            element.text = '{0}'.format(str(self._norm))
-
-
-    def create_num_flushes_subelement(self):
-
-        if not self._num_flushes is None:
-            element = ET.SubElement(self._cmfd_file, "num_flushes")
-            element.text = '{0}'.format(str(self._num_flushes))
-
+            element.text = str(self._norm)
 
-    def create_power_monitor_subelement(self):
-
-        if not self._power_monitor is None:
+    def _create_power_monitor_subelement(self):
+        if self._power_monitor is not None:
             element = ET.SubElement(self._cmfd_file, "power_monitor")
-            element.text = '{0}'.format(str(self._power_monitor).lower())
-
+            element.text = str(self._power_monitor).lower()
 
-    def create_run_adjoint_subelement(self):
-
-        if not self._run_adjoint is None:
+    def _create_run_adjoint_subelement(self):
+        if self._run_adjoint is not None:
             element = ET.SubElement(self._cmfd_file, "run_adjoint")
-            element.text = '{0}'.format(str(self._run_adjoint).lower())
-
-
-    def create_write_matrices_subelement(self):
-
-        if not self._write_matrices is None:
+            element.text = str(self._run_adjoint).lower()
+
+    def _create_shift_subelement(self):
+        if self._shift is not None:
+            element = ET.SubElement(self._shift, "shift")
+            element.text = str(self._shift)
+
+    def _create_spectral_subelement(self):
+        if self._spectral is not None:
+            element = ET.SubElement(self._spectral, "spectral")
+            element.text = str(self._spectral)
+
+    def _create_stol_subelement(self):
+        if self._stol is not None:
+            element = ET.SubElement(self._stol, "stol")
+            element.text = str(self._stol)
+
+    def _create_tally_reset_subelement(self):
+        if self._tally_reset is not None:
+            element = ET.SubElement(self._tally_reset, "tally_reset")
+            element.text = ' '.join(map(str, self._tally_reset))
+
+    def _create_write_matrices_subelement(self):
+        if self._write_matrices is not None:
             element = ET.SubElement(self._cmfd_file, "write_matrices")
-            element.text = '{0}'.format(str(self._write_matrices).lower())
-
+            element.text = str(self._write_matrices).lower()
 
     def export_to_xml(self):
-
-        self.create_active_flush_subelement()
-        self.create_begin_subelement()
-        self.create_display_subelement()
-        self.create_feedback_subelement()
-        self.create_inactive_subelement()
-        self.create_inactive_flush_subelement()
-        self.create_mesh_subelement()
-        self.create_norm_subelement()
-        self.create_num_flushes_subelement()
-        self.create_power_monitor_subelement()
-        self.create_run_adjoint_subelement()
-        self.create_write_matrices_subelement()
+        """Create a cmfd.xml file using the class data that can be used for an OpenMC
```

Comment:
Hmm, I see this is a little confusing. I meant that the cmfd.xml file can be used for OpenMC. Can you suggest a rewording?